src/utils/getFiltredPeople.ts

```diff
@@ -0,0 +1,33 @@
+import { Person } from '../types/Person';
+
+export const getFiltredPeople = (
+  people: Person[],
+  searchParams: URLSearchParams,
+) => {
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex') || '';
+  const centuries = searchParams.getAll('centuries');
```

Comment:
You did the same thing here as you did upper inside your component. Maybe it'll be better to pass to the function arguments `query`, `sex`, and `centuries` instead of the whole `searchParams`?